modules/hostap/Kconfig

```diff
@@ -136,9 +136,10 @@ config WPA_SUPP_CRYPTO_LEGACY
 	select MBEDTLS_ECP_C
 	select MBEDTLS_CTR_DRBG_C
 	select MBEDTLS_PK_C
+	select MBEDTLS_PK_PARSE_C
```

Comment:
Is this an unrelated change? If so, can you please move to a new commit with the commit log explaining the reason?

Reply:
Hi,

in the commit message I have written:

"Enable PK_PARSE_C as
hostap/src/crypto/crypto_mbedtls-ec.c
uses it."

does this suffice?

A different commit would make it less clear that this change is required because of the changing default values.

Reply:
Ah ok, may be we can modify that to:

```
Enable PK_PARSE_C explicitly (used by `hostap/src/crypto/crypto_mbedtls-ec.c`) now that legacy is disabled by default.
```

Reply:
will do :)

Reply:
done